includes/MirahezeMagicHooks.php

```diff
@@ -180,6 +180,22 @@ public static function onMessageCacheGet( &$lcKey ) {
 			'centralauth-login-error-locked',
 			'snapwikiskin',
 			'skinname-snapwikiskin',
+			"miraheze-group-checkuser": "Checkusers",
```

Comment:
The messages here should not be prefixed by `miraheze-` and you aren't adding values. Plus this isn't JSON anymore.

Reply:
Oh, right. Okay, that makes sense. Thanks. :)

Reply:
This still needs to use single quotes, and remove the `: "<text>"` stuff.